projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-settings.php

```diff
@@ -26,6 +27,10 @@ function test_can_write_settings() {
 
 		$updated_settings = Settings::get_settings();
 
+		// reset original value.
+		$settings['dequeue_max_bytes'] = $dequeue_max_bytes;
+		Settings::update_settings( $settings );
```

Comment:
This one is ok because there's no assertion in between changing the setting and resetting it.